Tests/SwiftyRequestTests/SwiftyRequestTests.swift

```diff
@@ -140,6 +140,7 @@ class SwiftyRequestTests: XCTestCase {
         let expectation = self.expectation(description: "Data Echoed Back")
 
         let request = RestRequest(method: .get, url: echoURLSecure, containsSelfSignedCert: true)
+        request.pinnedCertificateName = ""
```

Comment:
Please can we leave this test alone, add a new `testPinnedCertificate()` and test with a real pinned certificate? Or is that not possible?